crypto/obj/objects.go

```diff
@@ -376,7 +376,7 @@ func writeNumbers(path string, objs *objects) error {
 
 func clangFormat(input string) (string, error) {
 	var b bytes.Buffer
-	cmd := exec.Command("clang-format")
+	cmd := exec.Command("clang-format", "--style=Google")
```

Comment:
Do we need this? Our clang-format has `BasedOnStyle: Google`

Reply:
Without this option, the CI fails here due to it not recognizing the "InsertBraces" key I added to .clang-format. That option was only added in clang-format v15.